src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue

```diff
@@ -620,8 +620,12 @@ export default {
 		margin: 4px $icon-margin;
 	}
 
-	&__input:focus-visible + &__content {
-		outline: 2px solid var(--color-primary-element) !important;
+	&__input:focus-visible + &__content,
+	&__content:active,
+	&__input:focus-within + &__content {
```

Comment:
Are the `:active` and `:focus-within` pseudo-classes necessary? As now the borders are visible when just clicking with a mouse and stay until clicking elsewhere

Reply:
in my opinion we need it. without: 
![Peek 2023-10-30 18-46](https://github.com/nextcloud-libraries/nextcloud-vue/assets/6078378/0d78fcd3-4c61-4455-8bc3-70d44ad8d31e)

Reply:
I agree with @Pytal, I think it is a bit weird to have to focus outline also on click, we do not have it for buttons or other input elements.

Reply:
Was previously discussed with @michaelnissenbaum that only keyboard focus required focus borders but if you think this is better then fine with me :)